sonic-thermalctld/scripts/thermalctld

```diff
@@ -607,6 +607,9 @@ class ThermalMonitor(ProcessTaskBase):
 class ThermalControlDaemon(daemon_base.DaemonBase):
     # Interval to run thermal control logic
     INTERVAL = 60
+    RUN_POLICY_ELAPSE_THRESHOLD = 30
```

Comment:
This constant could be more descriptive and also include units, e.g., `RUN_POLICY_WARN_THRESHOLD_SECS`

Technically speaking, the other interval-related constants could use to have units added, also, but this can be saved for a separate PR.